packages/realm/src/assert.ts

```diff
@@ -71,14 +71,14 @@ assert.boolean = (value: unknown, target?: string): asserts value is boolean =>
   }
 };
 
-assert.bigInt = (value: unknown, target?: string): asserts value is bigint => {
-  if (typeof value !== "bigint") {
+assert.bigInt = (value: unknown, target?: string): asserts value is binding.Int64 => {
```

Comment:
I wonder if it makes sense to update this name as well?
```suggestion
assert.int64 = (value: unknown, target?: string): asserts value is binding.Int64 => {
```

Reply:
Hard to say since they aren't used for anything at the moment. I think it depends on how they end up being used. Maybe it would be best to just remove them for now, and we can re-add when needed?

Reply:
That'd be fine by me.